ethminer/MinerAux.h

```diff
@@ -317,7 +324,14 @@ class MinerCLI
 			}
 		else if ((arg == "--cl-local-work" || arg == "--cuda-block-size") && i + 1 < argc)
 			try {
-				m_localWorkSize = stol(argv[++i]);
+				if (arg == "--cl-local-work")
+				{
+					m_localWorkSize_CL = stol(argv[++i]);
```

Comment:
Good, 2 separate vars... name them for what they are.. so they kind of match the arg symbol.

ie. m_localWorkSize for CL, and m_blockSize for CUDA.

If you split the outer if, you won't need this inner if.